Corona-Warn-App/src/test/java/de/rki/coronawarnapp/util/formatter/FormatterStatisticsHelperTest.kt

```diff
@@ -0,0 +1,72 @@
+package de.rki.coronawarnapp.util.formatter
+
+import android.content.Context
+import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.contactdiary.util.getLocale
+import io.kotest.matchers.shouldBe
+import io.mockk.MockKAnnotations
+import io.mockk.every
+import io.mockk.impl.annotations.MockK
+import io.mockk.mockkStatic
+import io.mockk.unmockkAll
+import org.joda.time.LocalDate
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import java.util.Locale
+
+class FormatterStatisticsHelperTest {
+
+    @MockK
+    private lateinit var context: Context
+
+    private val today = LocalDate()
+    private val yesterday = LocalDate().minusDays(1)
+
+    @Before
+    fun setUp() {
+        MockKAnnotations.init(this)
+        mockkStatic("de.rki.coronawarnapp.contactdiary.util.ContactDiaryExtensionsKt")
+        with(context) {
+            every { context.getLocale() } returns Locale.GERMANY
+            every { getString(R.string.statistics_primary_value_today) } returns R.string.statistics_primary_value_today.toString()
+            every { getString(R.string.statistics_primary_value_yesterday) } returns R.string.statistics_primary_value_yesterday.toString()
+            every { getString(R.string.statistics_primary_value_until_today) } returns R.string.statistics_primary_value_until_today.toString()
+            every { getString(R.string.statistics_primary_value_until_yesterday) } returns R.string.statistics_primary_value_until_yesterday.toString()
+        }
+    }
+
+    @Test
+    fun `formatStatisticsDate test`() {
+        formatStatisticsDate(
+            context = context,
+            localDate = today
+        ) shouldBe context.getString(R.string.statistics_primary_value_today)
```

Comment:
I don't think, it is really testing what you want. if you check the output of this function it will be "2131886709".
Basically taking the resource Id value and converting it to a String. I would recommend to hard code the expected result if you are testing for German locale so you what to expect.

Reply:
I'm testing here only if methods use the right string resources, I don't need the real value. Like for example in `FormatterSubmissionHelperTest`

Reply:
I also think that hardcoded values are better because then you have real values e.g. in test error logs instead of just numbers.

Reply:
updated for better readability